src/java.base/share/classes/java/lang/Class.java

```diff
@@ -423,45 +424,52 @@ private static Class<?> forName(String className, Class<?> caller)
     /**
      * Returns the {@code Class} object associated with the class or
      * interface with the given string name, using the given class loader.
```

Comment:
Should we update the summary to `... associated with the class or interface or array with the given string name...`

Reply:
I want to keep it as is.  The first paragraph talks about a class or interface and the other paragraphs talk about array classes, primitive types and hidden classes.